gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
```

Comment:
This has been in PR so long Gradle is now up to 8.5 🙃

Reply:
make that 8.6

Reply:
Updated to 8.6